knowledge-base/cannot-implicitly-convert-type-telerik.reporting.xxxreportsource-to-telerik.reportviewer.html5.webforms.reportsource-error.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Cannot implicitly convert type 'Telerik.Reporting.XXXReportSource' to 'Telerik.ReportViewer.Html5.WebForms.ReportSource' error when using HTML5 WebForms Report Viewer
+description: Cannot implicitly convert type 'Telerik.Reporting.XXXReportSource' to 'Telerik.ReportViewer.Html5.WebForms.ReportSource' error when using HTML5 WebForms Report Viewer.
+type: how-to
+page_title: Cannot implicitly convert type 'Telerik.Reporting.XXXReportSource' to 'Telerik.ReportViewer.Html5.WebForms.ReportSource' error when using HTML5 WebForms Report Viewer
```

Comment:
Use something around the cause of this error for page_title. It will make it shorter and in the same time will add more keywords in terms of SEO